src/Analysers/AttributeAnnotationFactory.php

```diff
@@ -74,6 +74,22 @@ public function build(\Reflector $reflector, Context $context): array
                     }
                 }
             }
+
+            if ($reflector instanceof \ReflectionClass) {
+                foreach ($reflector->getReflectionConstants() as $rc) {
+                    foreach ($rc->getAttributes(Property::class) as $attribute) {
+                        $instance = $attribute->newInstance();
+                        if (Generator::isDefault($instance->property)) {
+                            $instance->property = $rc->getName();
+                        }
+                        if (Generator::isDefault($instance->type)) {
+                            $instance->type = 'const';
+                            $instance->const = [$rc->getValue()];
```

Comment:
Reading http://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.6.1.3 I would think this should be a plain value rather than an array?